openqaoa/workflows/parameters/qaoa_parameters.py

```diff
@@ -288,7 +288,8 @@ def __init__(self,
                  hess_options: dict = None,
                  optimization_progress: bool = False,
                  cost_progress: bool = True,
-                 parameter_log: bool = True):
+                 parameter_log: bool = True, 
+                 save_intermediate: bool = False):
```

Comment:
The parameter is missing from the docstring of the classical optimiser

Reply:
Added in the latest push